assets/common/src/model_deployment/deploy.py

```diff
@@ -255,11 +261,17 @@ def main(args):
     """Run main function."""
     ml_client = get_ml_client()
     # get registered model id
-    model_info = {}
-    with open(args.registration_details) as f:
-        model_info = json.load(f)
-    model_id = model_info['id']
-    model_name = model_info['name']
+    if args.registration_details:
+        model_info = {}
+        with open(args.registration_details) as f:
+            model_info = json.load(f)
+        model_id = model_info["id"]
+        model_name = model_info["name"]
+    elif args.model_id:
+        model_id = str(args.model_id)
+        model_name = model_id.split("/")[-3]
+    else:
+        raise Exception("Arguments model_id and registration_details both are missing.")
```

Comment:
Are we integrating logs in a different PR?

Reply:
Yes sasi. PR containing logs has lots of changes , so that is in different pr